assets/src/scss/style.scss

```diff
@@ -79,6 +79,9 @@ Text Domain: planet4-master-theme
 // CSS variables
 @import "base/css-variables";
 
+// Posts List Block
+@import "blocks/PostsListBlock";
```

Comment:
Why do we not add this to the [blocks style file](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/blocks/styles/blocks.scss) instead?

Reply:
Oh my bad, I had started working on this before Sagar merged the code to move blocks to the master theme, so let me change that.

Reply:
Maybe it actually makes more sense to do it this way 🤔 I'm not sure we need a separate style file for blocks, also it seems strange to have some styles outside of the `scss` folder